src/Http/Livewire/LivewireDatatable.php

```diff
@@ -1686,10 +1691,10 @@ public function getExportResultsSet()
             })->get(),
             true
         )->map(function ($item) {
-            return collect($this->columns())->reject(function ($value, $key) {
-                return $value->preventExport == true || $value->hidden == true;
-            })->mapWithKeys(function ($value, $key) use ($item) {
-                return [$value->label ?? $value->name => $item->{$value->name}];
+            return collect($this->columns)->reject(function ($value) {
```

Comment:
I have reverted this to get columns from ```$this->columns``` rather than ```$this->columns()```
Otherwise the export does not respect which fields are shown/hidden.

```$this->columns()``` will respect the hard-coded column definitions, whereas ```$this->columns``` respects the dynamic user-selected shown/hidden columns.

In my tables I want the users to be able to export the columns they have selected, but if the other behaviour is preferred then we should give the ability to change behaviour with a config variable